providers/gce/gce_instances.go

```diff
@@ -551,19 +551,24 @@ func (g *Cloud) AliasRangesByProviderID(providerID string) (cidrs []string, err
 		for _, r := range networkInterface.AliasIpRanges {
```

Comment:
Are there any other callers for AliasRangesByProviderID? Can this function be removed?

Reply:
Yes, https://github.com/kubernetes/cloud-provider-gcp/blob/master/pkg/controller/nodeipam/ipam/adapter.go#L70. Cannot remove the function until remove this usage.